clients/csharp-dotnet-core/voice-assistant-test/README.md

```diff
@@ -118,7 +118,7 @@ Here is the full list:
 >`JSON string | optional | null | [{"PropertyKey", "PropertyValue"}]`. A JSON string that is an array of pairs of two string values, used for custom settings of the Speech Service. Each pair results in a call to [DialogServiceConfig.SetServiceProperty(String, String, ServicePropertyChannel)](https://docs.microsoft.com/en-us/dotnet/api/microsoft.cognitiveservices.speech.dialog.dialogserviceconfig.setserviceproperty?view=azure-dotnet#Microsoft_CognitiveServices_Speech_Dialog_DialogServiceConfig_SetServiceProperty_System_String_System_String_Microsoft_CognitiveServices_Speech_ServicePropertyChannel_), where ServicePropertyChannel is set to ServicePropertyChannel.UriQueryParameter. Or the equivalent method on CustomCommandsConfig for custom command applications. For more detail, see the section [Custom Settings](#custom-settings)
 
 >#### [RealTimeAudio](#Measuring-User-Perceived-Latency)
->`bool | optional | false | true or false`. The default behavior of PushAudioInputStream is to send the first few seconds of audio as fast as possible to accomodate for short utterances. Following audio is throttled back to prevent client from spamming the service too fast but this throttled speed is faster than real-time microphone input. Set this optional flag to true to send audio at real-time (x1) speed. If this flag is set to true, the app config [Timeout](#Timeout) should be larger than the duration of the longest WAV file in the test.
+>`bool | optional | false | true or false`. The default behavior of PushAudioInputStream is to send the first few seconds of audio as fast as possible to accomodate for short utterances. Following audio is throttled back to prevent client from spamming the service too fast but this throttled speed is faster than real-time microphone input. Set this optional flag to true to send audio at real-time (x1) speed. If this flag is set to true, the app config [Timeout](#Timeout) should be larger than the duration of the longest WAV file in the test. For more information see [Measuring User Perceived Latency](#Measuring-User-Perceived-Latency) section.
 

```

Comment:
Lower case only in # links measuring-user-perceived-latency

Reply:
Turns out you don't need lower case

Reply:
Per, sajjad, no need for lower case

Reply:
You don't, but the Azure docs standard is lower case. Now as this is GitHub and not azure docs you can decide.